lstchain/io/io.py

```diff
@@ -50,12 +50,12 @@
     'write_calibration_data',
     'read_dl2_to_pyirf',
     'read_dl2_params',
-    'extract_observation_time',
-    'merge_dl2_runs'
```

Comment:
`'merge_dl2_runs'` is still present in the file

Reply:
> `'merge_dl2_runs'` is still present in the file

Should I remove?

Reply:
you have removed it in your PR, but the function is still included in the `io.py`, therefore you should include it again in the `__all__` definition.